webapp/src/components/testing/Testing.vue

```diff
@@ -30,39 +30,23 @@
                 "
                 class="shadow shadow-disco-cyan"
               >
-                <template #title>
-                  {{ taskTitle(metadata.taskID) }}
+                <template #title> {{ taskTitle(infos.taskID) }} </template>
+                <template #icon>
+                  <button @click="removeModel(id)"><Bin2Icon /></button>
```

Comment:
Very nice! A small hover feedback would be great
```suggestion
                  <button 
                    @click="removeModel(id)"
                    class="hover:text-red-500 transition duration-200"
                  >
                    <Bin2Icon />
                  </button>
```

Reply:
hoo, that's so fancy, thanks for the tip!